### ISSUE TITLE
Read page duration limits. 

### ISSUE THREAD

**Issue by Zeyadas** [koreader/koreader#8266]
I mostly read pdfs, and usually there is a blank page(s) in my files (books), the minimum read page duration limit Is 3 seconds. If I skipped these blanket pages koreader wouldn't (count) it in pages read and in statistics.
So simply I suggest if you can make the minimum limit to (zero) an option. 
Thanks.

**Comment by Galunid** [comment]
You can just set it to 0? `4th menu -> Reading statistics -> Settings -> Read page duration limits`
Unless I misunderstood what you want

**Comment by hius07** [comment]
https://github.com/koreader/koreader/blob/a21b5be64a25f33d3d8194b8b1282805f9a990c2/plugins/statistics.koplugin/main.lua#L906

**Comment by Frenzie** [comment]
So let's just change that to 0? Or is there a reason not to.

**Comment by Galunid** [comment]
https://github.com/koreader/koreader/blob/a21b5be64a25f33d3d8194b8b1282805f9a990c2/plugins/statistics.koplugin/main.lua#L2203-L2210
Is the only place where it's used (`self.settings.min_sec`), and it doesn't look like it'll break anything.

**Comment by Zeyadas** [comment]
Sorry for posting here again.
That's an 18 pages document I skipped all the pages after setting the read page duration limit to 0, and koreader didn't count all the pages.
Tested on my both devices (android tablet and android phone). 
It was working perfectly but I didn't notice when this started to happen, I mean which version. 

Now I'm on the 2021.21.1 version. 
![Screenshot_20220106-152039](https://user-images.githubusercontent.com/70109540/148382403-e5249554-546b-4c69-9f14-b786c82744f6.png)
